Dfc.CourseDirectory.FindACourseApi/Resources/AzureDevops/GitVersion.yml

```diff
@@ -1,4 +1,67 @@
 mode: ContinuousDelivery
 branches:
   master:
-    regex: ^main
\ No newline at end of file
+    regex: ^master$|^main$
```

Comment:
Isn't master dead?

Reply:
yeh! it's a generic script and I'm far too lazy to adjust it 👍